app/models/user.rb

```diff
@@ -213,6 +212,10 @@ def suspended?
     suspended_at.present?
   end
 
+  def active?
```

Comment:
Consider not defining a new method and relying on `active_for_authentication?` in the tests. That will prove that Devise won't log the user in, which seems like a great test.